src/Megamix/Region.cpp

```diff
@@ -292,6 +292,96 @@ namespace Region {
         }
     }
 
+    // Various locations used for the Tickflow Command flow
+
+    u32 TickflowCommandsSwitch() {
+        switch (region) {
+            case JP:
+                return 0x25e054;
+            case US:
+            case EU:
+                return 0x25c3c0;
+            case KR:
+                return 0x25c398;
+            default:
+                return 0;
+        }
+    }
+
+    u32 TickflowCommandsEnd() {
+        switch (region) {
+            case JP:
+                return 0x262eac;
+            case US:
+            case EU:
+                return 0x2613cc;
+            case KR:
+                return 0x2613a4;
+            default:
+                return 0;
+        }
+    }
+
+    // Location of the code for async_sub (fixes custom commands)
+
+    u32 TickflowAsyncSubLocation() {
+        switch (region) {
+            case JP:
+                return 0x25e338;
+            case US:
+            case EU:
+                return 0x25c6c0;
+            case KR:
+                return 0x25c698;
+            default:
+                return 0;
+        }
+    }
+
+    // Locations of global variables
+
+    u32 GlobalSaveDataPointer(){
+        switch (region) {
+            case JP:
```

Comment:
if you have no value for JP please move it to the bottom of the switch-case along with `default`. same applies to  `GlobalInputManagerPointer` and `GlobalFileManagerPointer`.